qiskit_experiments/library/characterization/resonator_spectroscopy.py

```diff
@@ -53,12 +53,24 @@ class ResonatorSpectroscopy(Spectroscopy):
             c: 1/════════════════════╩═
                                      0
 
-        Side note: when doing readout resonator spectroscopy, each measured IQ point has a
+        Side note 1: when doing readout resonator spectroscopy, each measured IQ point has a
```

Comment:
There is a `# section: note` that might be a good fit for these notes. [FineDragAnalysis](https://qiskit.org/documentation/experiments/dev/stubs/qiskit_experiments.library.characterization.FineDragAnalysis.html#qiskit_experiments.library.characterization.FineDragAnalysis) is one example of a class using it. I am not sure about having two notes though.